libs/entry-components/styles/modules/components/slide-togglers/_generator.scss

```diff
@@ -0,0 +1,33 @@
+/* stylelint-disable scss/selector-no-redundant-nesting-selector */
+@use 'sass:map';
+@mixin generate-from($theme) {
+	.mat-mdc-slide-toggle {
+		$enabled-states: 'enabled', 'enabled:focus:not(:active)', 'enabled:active', 'enabled:hover:not(:focus):not(:active)';
+
+		$track-on-background: map.get($theme, 'general', 'slide-togglers', 'on', 'track', 'background');
```

Comment:
To reduce amount of duplication and strings count, you can say:
$toggle: map.get($theme, 'general', 'toggle');
$track-on-background: map.get($toggle, 'on', 'track');
etc.